README.md

```diff
@@ -54,275 +55,250 @@ limitations under the License.
+In order to celebrate Transformers 100,000 stars, we wanted to put the spotlight on the
+community with the [awesome-transformers](./awesome-transformers.md) page which lists 100
+incredible projects built with Transformers.
 
-```
-git clone https://github.com/huggingface/transformers.git
-cd transformers
-pip install .
-```
+If you own or use a project that you believe should be part of the list, please open a PR to add it!
 
-### With conda
+## Example models
 
-🤗 Transformers can be installed using conda as follows:
+You can test most of our models directly on their [Hub model pages](https://huggingface.co/models).
 
-```shell script
-conda install conda-forge::transformers
-```
+Expand each modality below to see a few example models for various use cases.
 
-> **_NOTE:_** Installing `transformers` from the `huggingface` channel is deprecated.
+<details>
+<summary>Audio</summary>
 
-Follow the installation pages of Flax, PyTorch or TensorFlow to see how to install them with conda.
+- Audio classification with [Whisper](https://huggingface.co/openai/whisper-large-v3-turbo)
+- Automatic speech recognition with [Moonshine](https://huggingface.co/UsefulSensors/moonshine)
+- Keyword spotting with [Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
+- Speech to speech generation with [Moshi](https://huggingface.co/kyutai/moshiko-pytorch-bf16)
+- Text to audio with [MusicGen](https://huggingface.co/facebook/musicgen-large)
+- Text to speech with [Bark](https://huggingface.co/suno/bark)
 
-> **_NOTE:_**  On Windows, you may be prompted to activate Developer Mode in order to benefit from caching. If this is not an option for you, please let us know in [this issue](https://github.com/huggingface/huggingface_hub/issues/1062).
+</details>
 
-## Model architectures
+<details>
+<summary>Computer vision</summary>
 
-**[All the model checkpoints](https://huggingface.co/models)** provided by 🤗 Transformers are seamlessly integrated from the huggingface.co [model hub](https://huggingface.co/models), where they are uploaded directly by [users](https://huggingface.co/users) and [organizations](https://huggingface.co/organizations).
+- Automatic mask generation with [SAM](https://huggingface.co/facebook/sam-vit-base)
+- Depth estimation with [DepthPro](https://huggingface.co/apple/DepthPro-hf)
+- Image classification with [DINO v2](https://huggingface.co/facebook/dinov2-base)
+- Keypoint detection with [SuperGlue](https://huggingface.co/magic-leap-community/superglue_outdoor)
+- Object detection with [RT-DETRv2](https://huggingface.co/PekingU/rtdetr_v2_r50vd)
+- Universal segmentation with [OneFormer](https://huggingface.co/shi-labs/oneformer_ade20k_swin_large)
+- Video classification with [VideoMAE](https://huggingface.co/MCG-NJU/videomae-large)
```

Comment:
cc @qubvel @merveenoyan in case you see better/others that would fit here

Reply:
```
 - Pose Estimation with [VitPose](https://huggingface.co/usyd-community/vitpose-base-simple)
 - Keypoint matching with [SuperGlue](https://huggingface.co/magic-leap-community/superglue)
```